openshift/generate-manifests.sh

```diff
@@ -41,7 +57,7 @@ mkdir -p "${TMP_ROOT}/openshift"
 cp -a "${REPO_ROOT}/openshift/kustomize" "${TMP_ROOT}/openshift/kustomize"
 
 # Override OPENSHIFT-NAMESPACE to ${NAMESPACE}
-find "${TMP_ROOT}" -name "*.yaml" -exec sed -i "s/OPENSHIFT-NAMESPACE/${NAMESPACE}/g" {} \;
+find "${TMP_ROOT}" -name "*.yaml" -exec "$SED_CMD" -i "s/OPENSHIFT-NAMESPACE/${NAMESPACE}/g" {} \;
```

Comment:
```suggestion
if [[ "$OSTYPE" == "darwin"* ]]; then
  find "${TMP_ROOT}" -name "*.yaml" -exec sed -i "s/OPENSHIFT-NAMESPACE/${NAMESPACE}/g" {} \;
else
  find "${TMP_ROOT}" -name "*.yaml" -exec sed -i "" "s/OPENSHIFT-NAMESPACE/${NAMESPACE}/g" {} \;
fi
```

Reply:
Because blank arguments are a bit weird, I went the safe route and just did the line twice. Although, you could instead, just supply a `.tmp` suffix regardless, because this is all done in a temp directory.

Reply:
Aha. TIL. Updated.